src/registry/views/component-preview.pug

```diff
@@ -12,3 +12,4 @@ html
       oc.conf.templates = oc.conf.templates || [];
       oc.conf.templates = oc.conf.templates.concat(!{JSON.stringify(templates)});
     script(src=baseUrl+'oc-client/client.js')
+    | !{liveReload}
```

Comment:
What about when this view is used in the UI? Should we put this here (with a localhost url) only if the registry is on local mode?

Reply:
I'm doing that here -> https://github.com/opentable/oc/pull/635/files#diff-e4f8fa94a02abcf0d568f89a410ab48eR16 (if that view is used on the UI when the registry is not on local mode, nothing is printed)